spec/operators/every-spec.ts

```diff
@@ -167,7 +167,7 @@ describe('Observable.prototype.every', () => {
     const source = Observable.of(3);
     const expected = '#';
 
-    function faultyPredicate(x) {
+    function faultyPredicate(x: never) {
```

Comment:
I was just using `never` since the value is never used in the actual method.  Regardless either way works. 👍